Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM maven:3.6.2-jdk-11-slim AS build
 WORKDIR /usr/app
 COPY src ./src
 COPY pom.xml .
+COPY internship-cloud/*.json ./internship-project.json
```

Comment:
Why are you coping it to the build container first? Copy it only to the final container

Reply:
fixed